air/src/options.rs

```diff
@@ -63,7 +63,7 @@ pub struct ProofOptions {
     grinding_factor: u8,
     field_extension: FieldExtension,
     fri_folding_factor: u8,
-    fri_max_remainder_size: u8, // stored as power of 2
```

Comment:
Should we name this `fri_max_remainder_degree`? I think this would better reflect what it actually is. We could still store it as a power of two, but now we'd be storing `log2(max_degree + 1)`.

Reply:
Makes sense.